site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -157,8 +157,8 @@ private function editPage(Gradeable $gradeable, $semester, $course, $nav_tab = 0
 
         $type_string = 'UNKNOWN';
         if ($gradeable->getType() === GradeableType::ELECTRONIC_FILE) {
-            if ($gradeable->isScannedExam()) {
-                $type_string = self::gradeable_type_strings['electronic_exam'];
+            if ($gradeable->isStudentView() && $gradeable->isStudentViewAfterGrades()) {
```

Comment:
I think it would make sense for a function in the gradeable model that returns this so people know what this combination of checks is supposed to represent

something like:
```
isBulkUpload(){
   return $this->isStudentView() && $this->isStudentViewAfterGrades();
}
```